### PR TITLE
Additional tags for the Rank Math SEO plugin

### PR DESCRIPTION
**Pull request by MyThemeShop**
This PR adds options to enable/disable specific data from the Rank Math SEO plugin:
https://i.mythemeshop.com/wpKOJZ

### CODE REVIEW

**Comment by MohammedKaludi** [comment]
@MyThemeShop Thank you for the PR, will check and get back to you.

File: templates/features.php [hunk 1]
```diff
@@ -5853,10 +5852,10 @@ function ampforwp_generate_canonical(){
 		$opts = $All_in_One_SEO_Pack->get_current_options( array(), 'aiosp' );
 		$canonical = $opts['aiosp_custom_link'];
 	}
-	elseif (defined( 'RANK_MATH_FILE' ) && 'rank_math' == ampforwp_get_setting('ampforwp-seo-selection') ) {
+	elseif (defined( 'RANK_MATH_FILE' ) && 'rank_math' == ampforwp_get_setting('ampforwp-seo-selection') && isset( $redux_builder_amp['ampforwp-seo-rank_math-canonical'] ) && $redux_builder_amp['ampforwp-seo-rank_math-canonical'] ) {
```
Comment (MARQAS) [inline templates/features.php]: Use `ampforwp_get_setting`

File: templates/features.php [hunk 2]
```diff
@@ -6487,14 +6486,32 @@ function ampforwp_get_weglot_url(){
...
 			return;
 		}
+
 		// Remove Canonical & Title Tag added by the Rank Math plugin.
 		remove_all_actions( 'rank_math/head', 20 );
 		remove_all_actions( 'rank_math/head', 1 );
+
+		global $redux_builder_amp;
+		// Remove meta tags added by the Rank Math plugin.
+		if ( ! isset( $redux_builder_amp['ampforwp-seo-rank_math-meta'] ) || ! $redux_builder_amp['ampforwp-seo-rank_math-meta'] ) {
```
Comment (MARQAS) [inline templates/features.php]: Use `ampforwp_get_setting`

File: templates/features.php [hunk 3]
```diff
@@ -6487,14 +6486,32 @@ function ampforwp_get_weglot_url(){
...
+			$json_ld_data = isset( $wp_filter['rank_math/json_ld'] ) ? $wp_filter['rank_math/json_ld'] : '';
+			remove_all_actions( 'rank_math/opengraph/facebook' );
+			remove_all_actions( 'rank_math/opengraph/twitter' );
+			add_filter( 'rank_math/frontend/robots', function() {
+				return [];
+			});
+		}
+
+		// Remove ld+json data added by the Rank math plugin.
+		if ( ! isset( $redux_builder_amp['ampforwp-seo-rank_math-schema'] ) || ! $redux_builder_amp['ampforwp-seo-rank_math-schema'] ) {
```
Comment (MARQAS) [inline templates/features.php]: Use `ampforwp_get_setting`

**Comment by MARQAS** [comment]
@MyThemeShop, It is working properly but need some changes, I have mentioned them, please make changes accordingly.

**Comment by MARQAS** [comment]
@MyThemeShop Any update on this?

**Comment by MyThemeShop** [comment]
Hi @MARQAS, sorry for the unwanted delay.

Added related changes here:
https://github.com/ahmedkaludi/accelerated-mobile-pages/pull/3314

and closing this PR, as the new PR contains all the code.

**Comment by MARQAS** [comment]
Sure, I'll look into it tomorrow and will let you know.

**Pull request closed without merging**